Automattic/wp-calypso
# Gutenberg: update packages to latest versions

Resolves https://github.com/Automattic/wp-calypso/issues/27649

#### Changes proposed in this Pull Request

Updates the Gutenberg npm packages to their latest versions released on 11th of October.

#### Testing instructions

1. Navigate to http://calypso.localhost:3000/gutenberg/ and verify that the editor loads.
2. Smoke test the editor and check for regressions.
3. Calypso smoke test.

#### Testing jQuery removal from externals

I discussed this with @blowery and it should be safe to remove the jQuery from externals.

1. Make sure that there are no `import jQuery` in the Calypso codebase.
1. Make sure that we are not causing regressions for parts of Calypso that use jQuery via load-script.
2. Check [bundle sizes](http://iscalypsofastyet.com/branch?branch=update/gutenberg-packages-2).
3. Check jQuery 3 (editor sub-dep) and jQuery 1 (via load-script) interaction. For example, this can be done by using some functionality that triggers the jQuery 1 load, and then loading the editor shell.
